code/obj/machinery/door/door_timer.dm

```diff
@@ -310,6 +319,20 @@
 					logTheThing("station", usr, null, "sets off flashers from a door timer: [src] [log_loc(src)].")
 					return TRUE
 
+		if ("toggle-flusher")
+			for (var/obj/machinery/floorflusher/FF in range(30, src))
```

Comment:
30????????

Reply:
That is the range check used for the floorflusher on line 221, which is clarified on line 192, saying that "Cog2 places linked fixtures much further away from the timer than originally envisioned."
I am unsure if this still holds true, however.

Reply:
ok can be fixed later, as this will lead to same perf